wire-ios/Wire-iOS/Sources/Managers/Image/ImagePickerManager.swift

```diff
@@ -76,11 +86,12 @@ class ImagePickerManager: NSObject {
         return actionSheet
     }
 
-    private func getImage(fromSourceType sourceType: UIImagePickerController.SourceType) {
-        guard UIImagePickerController.isSourceTypeAvailable(sourceType),
-              let viewController else {
-                  return
-              }
+    private func getImage(
+        fromSourceType sourceType: UIImagePickerController.SourceType,
+        viewController: UIViewController,
+        popoverSourceView: UIView
```

Comment:
optional: We also have a type `PopoverPresentationControllerConfiguration`. Eventually we should be consistent  and either use `PopoverPresentationControllerConfiguration` or delete it, in my opinion.

Reply:
@caldrian I'll update this to use `PopoverPresentationControllerConfiguration`

Reply:
Using `PopoverPresentationControllerConfiguration` in c6a5c7bda430113f08e6cda75818b19e60766e12